test/dynamo/test_logging.py

```diff
@@ -25,19 +25,22 @@
     skipIfTorchDynamo,
     xfailIfS390X,
 )
-from torch.testing._internal.inductor_utils import HAS_CUDA
+from torch.testing._internal.inductor_utils import HAS_GPU, HAS_CUDA
 from torch.testing._internal.logging_utils import (
     LoggingTestCase,
     make_logging_test,
     make_settings_test,
 )
 
 
+requires_gpu = unittest.skipUnless(HAS_GPU, "requires gpu")
 requires_cuda = unittest.skipUnless(HAS_CUDA, "requires cuda")
 requires_distributed = functools.partial(
     unittest.skipIf, not dist.is_available(), "requires distributed"
 )
 
+device_type = torch.accelerator.current_accelerator().type if torch.accelerator.current_accelerator() \
+    is not None else 'cpu'
```

Comment:
```suggestion
device_type = acc.type if (acc := torch.accelerator.current_accelerator(True)) else "cpu"
```

Reply:
Thanks. I will return "cuda" by default as the original code is for cuda.